.drone/drone.yml

```diff
@@ -1099,8 +1099,3 @@ get:
   path: secret/data/common/loki_ci_autodeploy
 kind: secret
 name: deploy_config
----
-kind: signature
-hmac: d46edb356841403724d7a261f095fcbaf22e0e077e35e9b2e4c8bbad7aa4b77e
-
-...
```

Comment:
This was removed because I don't have the variables `DRONE_SERVER` and `DRONE_TOKEN` set. Any idea where I can find these values?

Reply:
Values can be found on the drone settings page. Updated the error message.